firmware/src/constants.h

```diff
@@ -88,25 +88,75 @@
  * @brief The Ja Rule message commands.
  */
 typedef enum {
-  ECHO = 0x80,  //!< Echo the data back
-  TX_DMX = 0x81,  //!< Transmit a DMX frame
+  ECHO = 0x80,  //!< Echo the data back. See @ref message_cmd_echo
+
```

Comment:
The random new space after echo. They don't seem to be grouped into logical groups, just randomly.